src/pages/Reviews.js

```diff
@@ -170,7 +178,7 @@ const Reviews = ({ productId }) => {
             <h3>{review.title}</h3>
             <p>Rating: {review.rating}</p>
             <p>Comment: {review.comment}</p>
-            {user && (
+            {user && (review.user === user.userId) && (
```

Comment:
Let's not show the update/delete buttons for reviews that don't belong to the current user; to reduce confusion.

Reply:
@akosasante here we are looking if id of the current user that we fetched in this component- in useEffect  lines 6-23 is matching review.user - the user key we've got from cookies?